client-data/tools/pencil/pencil.js

```diff
@@ -118,9 +118,11 @@
 		switch (nbr) {
 			case 0: //The first point in the line
 				//If there is no point, we have to start the line with a moveTo statement
-				npoint = { type: "M", values: [x, y] };
+				pts.push({ type: "M", values: [x, y] });
+				//Temporary first point so that clicks are shown and can be erased
+				npoint = { type: "L", values: [x, y] };
 				break;
-			case 1: //There is only one point.
+			case 2: //There are two points, however one of them might be temporary. Since it is a line of 0 length we can ignore it
```

Comment:
I don't understand this comment. Why *might* the point be temporary ?

Reply:
There should never be a case where there is only one point since if there are no points, two will be created. I guess there could be old boards that have only a single child that we might need to deal with.
I initially planned for the first point to be deleted later but found that to be impossible (we create the line and have two points -> we add another point and want to delete the previous point since it is no longer needed -> now we still have 3 points and keep deleting the previous one. Checking the type of the second point did not work as chrome set it to be a line every time).

Reply:
OK, thanks for the clear explanation !
If we are sure there can never be 1 point, then we should throw an error in that case, otherwise, we should probably just handle it as before.

Reply:
I will update it to ensure correct rendering of older boards